keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/UnknownBaseImage.kt

```diff
@@ -1,6 +1,7 @@
 package com.netflix.spinnaker.keel.bakery
 
 import com.netflix.spinnaker.keel.api.artifacts.BaseLabel
-import com.netflix.spinnaker.kork.exceptions.UserException
+import com.netflix.spinnaker.kork.exceptions.ConfigurationException
 
-class UnknownBaseImage(os: String, label: BaseLabel) : UserException("Could not identify base image for os $os and label $label")
+class UnknownBaseImage(os: String, label: BaseLabel) :
+  ConfigurationException("Could not identify base image for os $os and label $label")
```

Comment:
I'm not sure that's a `ConfigurationException` is it?

Reply:
Sub-class of `UserException` specifically for when the error is with something the user mis-configured. I went back and forth on using this type. I can change it to `UserException` if you think it's better.

Reply:
Sorry, I misread your comment. You're right. Will change to `SystemException`.